runtime/syntax/man.vim

```diff
@@ -5,18 +5,21 @@ if exists('b:current_syntax')
   finish
 endif
 
-syntax case  ignore
-syntax match manReference      display '[^()[:space:]]\+([0-9nx][a-z]*)'
-syntax match manSectionHeading display '^\S.*$'
-syntax match manTitle          display '^\%1l.*$'
-syntax match manSubHeading     display '^ \{3\}\S.*$'
-syntax match manOptionDesc     display '^\s\+\%(+\|-\)\S\+'
+" Not contained because sometimes, manpages contain garbage backspaced
+" characters. See Restricted Shell at bash(1)
```

Comment:
Very interesting. And yea, I missed out on all that.

I just bolded any backspaced character that does not have an underline before the backspace since as you said, it's unlikely for these characters to be in man pages and it's simple.